brefv/payloads/foxglove/Imu.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package foxglove;
```

Comment:
the foxglove package is not our own, but it comes from here: https://github.com/foxglove/schemas

For that reason, we can (actually should) not add our own message definitions here. I suggest we move this up one level in the hierarchy and treat it as a `brefv` message instead. Note that we still can (and should!) make use of other foxglove messages to build this message as you have already done!

Reply:
Fixed